shuup/core/utils/price_display.py

```diff
@@ -74,7 +74,7 @@ def _register(self):
 class PriceDisplayFilter(_ContextFilter):
     def __call__(self, context, item, quantity=1, include_taxes=None, allow_cache=True):
         key, val = context_cache.get_cached_value(
-            identifier=self.cache_identifier, item=item, context=context,
+            identifier=self.cache_identifier, item=item, context=context.get('request', context),
```

Comment:
Agree. Since it is not part of the initial task no need to merge this as part of this PR.